streamz/dataframe/aggregations.py

```diff
@@ -202,10 +203,10 @@ def diff_loc(dfs, new, window=None):
     """
     dfs = deque(dfs)
     dfs.append(new)
-    mx = max(df.index.max() for df in dfs)
-    mn = mx - window
+    mx = pd.Timestamp(max(df.index.max() for df in dfs))
+    mn = pd.Timestamp(mx) - window
```

Comment:
Isn't mx already a pd.Timestamp type, why pass it into another pd.Timestamp(...) type?

Reply:
Fixed